tests/test_secrets.py

```diff
@@ -131,7 +136,7 @@ async def test_permissions(ds, path, verb, data, user):
 
 @pytest.mark.asyncio
 async def test_set_secret(ds, use_actors_plugin):
-    cookies = {"ds_actor": ds.client.actor_cookie({"id": "admin"})}
+    cookies = {"ds_actor": actor_cookie(ds, {"id": "admin"})}
```

Comment:
Using this new feature in `datasette-test`:
- https://github.com/datasette/datasette-test/issues/5